src/lib/konnectors.js

```diff
@@ -41,7 +41,7 @@ export function addAccount (cozy, konnector, account) {
 export function fetchManifest (cozy, source) {
   return source
     ? cozy.fetchJSON('GET', `/konnectors/manifests?Source=${encodeURIComponent(source)}`)
-    : Promise.reject(new Error('Source konnector is unavailable'))
+    : Promise.reject(new Error('You must specify a source to be able to fetch the konnector manifest'))
```

Comment:
I personnally dislike adressing directly the user in error message, using _You_. Especially when the error message is targeted to the developer. What do you think about using something less personal like `A source must be provided to fetch the konnector manifest`

Reply:
Great advice, thanks !